Client-app/src/components/ClientProfile/ProfileEditModal.js

```diff
@@ -299,20 +299,23 @@ export default function BasicModal() {
                     label="Date of Birth"
                     value={dob}
                     onChange={handleUpdateDOB}
-                    renderInput={(params) => <TextField 
-                      className={classes.root}
-                      sx={{
-                        input: {
-                          background: "#262626",
-                          borderRadius: "5px",
-                        }
-                      }}
-                      InputLabelProps={{
-                        sx: {
-                          color: "var(--text-primary)",
-                        },
-                      }}
-                      {...params} />}
```

Comment:
Lots of the code are being auto-corrected on save because someone else (who edited this file previously) still didn't updated their VSCode Settings (quote: double quote, tabWidth: 2)